nUpdate.Shared/Updating/UpdateManager.cs

```diff
@@ -228,6 +228,20 @@ public CultureInfo LanguageCulture
         /// </summary>
         public int SearchTimeout { get; set; } = 10000;
 
+        /// <summary>
+        ///     Gets or sets a value indicating whether the nUpdate download the update-packages from uri-path of <see cref="UpdateConfigurationFileUri"/> 
```

Comment:
Please make sure that the grammar in the documentation fits in with the other comments.
It should be something like "Gets or sets a value indicating whether nUpdate should download the updates packages from the specified <see cref="UpdateConfigurationFileUri"/> instead of each <see cref="UpdateConfiguration.UpdatePackageUri"/> from the update configuration."